toolbox

```diff
@@ -529,15 +530,15 @@ list_containers()
     if ! output=$(echo "$containers" \
                   | sed "s/ \{2,\}/\t/g" 2>&3 \
                   | column \
-                            --separator $'\t' \
+                            --separator "$tab" \
                             --table \
                             --table-columns "CONTAINER ID,CONTAINER NAME,CREATED,STATUS,IMAGE NAME" 2>&3); then
         echo "$base_toolbox_command: failed to parse list of containers" >&2
         return 1
     fi
 
     if [ "$output" != "" ]; then
-        echo -e "${LBC}Containers created by toolbox${NC}"
+        echo "${LBC}Containers created by toolbox${NC}"
```

Comment:
I was trying to read up on why the `-e` doesn't matter. The `man` pages for both the bash built-in in `bash(1)` and the external binary in `echo(1)` say that when `-e` is in effect the `\0NNN` sequence is one of the recognized sequences. Wouldn't that cover this case?

Or am I reading it wrong? Another option is to use `printf`.

Reply:
First of all, sorry for the delay. Yes, you are correct, I just removed the -e flag and it seemed to work fine, but apparently I was incorrect. I pushed a new commit that uses printf.

Reply:
No need to apologize. :) Thanks for updating the patch.